### ISSUE TITLE
Unable to start the environment on vso-plan-westeurope

### ISSUE THREAD

**Issue by patrickbecker1983** [MicrosoftDocs/vsonline#333]
Hello,
every environment on vso-plan-westeurope is unable to start since yesterday. Likewise it is not possibile to create a new one. 

Tested with different Browsers on different places wiht different Azure/VSO Accounts.

Error Message: Connecting to environment XXX. Error: Unable to start the environment. Please try again in a few minutes.

**Comment by jkeech** [comment]
Sorry you are hitting this issue. We are investigating. Can you share one of your environment ids (or paste the url you see when you try to start the environment)?

**Comment by patrickbecker1983** [comment]
Thank you for your quick response.
Do you mean this one?
https://online.visualstudio.com/environment/6b129635-bac5-47cd-8e69-719b78b2d2fe

**Comment by jkeech** [comment]
Yes, thanks! That will help us look at what went wrong for your specific environment.

**Comment by patrickbecker1983** [comment]
Thanks,
my second env is althoug affected:
https://online.visualstudio.com/environment/cf7e9e4f-eb47-4acb-9304-f05c6b6f705e

**Comment by bennycornelissen** [comment]
Getting the same error message when trying to create an environment. Anything I can do to troubleshoot? I have no existing environments so no environment IDs to share.

**Comment by nikmd23** [comment]
@stevencl was having this issue as well.

**Comment by olivermt** [comment]
same for e0b07ca3-c48f-4e7c-9273-97a54cc27fa0 (and two others on same plan)

**Comment by eljog** [comment]
We've identified and fixed the issue that was preventing environments from starting/creating in the West Europe region. There was a traffic routing issue in our Kubernetes cluster that was preventing requests from completing successfully.
